lib/ui/settings/settings_page.dart

```diff
@@ -1,10 +1,12 @@
 import 'package:flutter/material.dart';
```

Comment:
i messed up and pushed UI code i was doing for settings on this PR...

Reply:
<img width="408" alt="image" src="https://user-images.githubusercontent.com/8041901/211968919-d0b4de45-fda8-4ca6-8d5a-2e3dbd50ea61.png">